src/crate/crash/command.py

```diff
@@ -369,6 +374,38 @@ def _show_tables(self, *args):
                       where schema_name not in ('sys','information_schema')""")
 
     @noargs_command
```

Comment:
decorator only required for commands that are called in the `_try_exec_cmd` method